products/vmware-horizon.md

```diff
@@ -19,60 +19,87 @@ releases:
     releaseDate: 2022-10-20
     eol: 2025-10-20
     support: 2026-10-20
+    latest: "2209"
```

Comment:
looks like all `latest` tags are almost same with `codename` tags so , lets remove codename tags ( all )
and change changelogtemplate to `"Horizon __RELEASE_CYCLE__  __LATEST__"`